library/src/main/java/com/smartnsoft/droid4me/download/gif/GifFrame.java

```diff
@@ -0,0 +1,50 @@
+package com.smartnsoft.droid4me.download.gif;
+
+/**
+ * @author Antoine Gerard
+ * @since 2017.10.18
+ */
+
+/**
+ * Inner model class housing metadata for each frame.
+ */
```

Comment:
Ces deux blocs doivent être regroupés. Exemple :

```
/**
 * Inner model class housing metadata for each frame.
 *
 * @author Antoine Gerard
 * @since 2017.10.18
 */
```